### ISSUE TITLE
Add swab type to settings page and cards in testing queue

### ISSUE THREAD

**Issue by aliciabeckett-gov** [CDCgov/prime-simplereport#358]
Blocked by CDCgov/prime-central#152 design updates

- [ ] Swab Type (similar to devices)
- [ ] Add swab to test queue cards too

See CDCgov/prime-central#106 to see updates about Swab Type

**Comment by aliciabeckett-gov** [comment]
Per the email from Riki at APHL, 445297001^Swab of internal nose^SC is the best default when we don't have a real answer.

In general seems like Specimen Source Site is the code next to the swab type. I can break this into a separate column in my spreadsheet if that's hellpful.

Everything you ever wanted to know about swabs: https://www.cdc.gov/flu/pdf/professionals/flu-specimen-collection-poster.pdf

**Comment by aliciabeckett-gov** [comment]
Initial Swab Type -> Device mapping. Most devices only have one swab type, but a couple have multiple options.
[Waived Tests Mapping LIVD 12-16.xlsx](https://github.com/CDCgov/prime-simplereport/files/5735863/Waived.Tests.Mapping.LIVD.12-16.xlsx)

**Comment by aliciabeckett-gov** [comment]
Duplicate of #384
